--- /usr/lib/python3.6/site-packages/kiwi/bootloader/template/isolinux.py    2019-07-22 21:16:22.459215665 +0000
+++ /usr/lib/python3.6/site-packages/kiwi/bootloader/template/isolinux.py 2019-07-22 21:16:46.171269970 +0000
@@ -178,10 +178,11 @@
         if terminal == 'serial':
             template_data += self.serial
             with_theme = False
-        if with_theme:
-            template_data += self.ui_theme
         else:
-            template_data += self.ui_plain
+            if with_theme:
+                template_data += self.ui_theme
+            else:
+                template_data += self.ui_plain
         template_data += self.menu_entry
         if failsafe:
             template_data += self.menu_entry_failsafe
